### ISSUE TITLE
Regenerating Project Files After Install Results In Error

### ISSUE THREAD

**Issue by CJRobertson** [AaronShea/BLUI#73]
I've been trying to install the BLUI plugin for a few days now and haven't had much luck. I've been following the READ ME exactly but when attempting to regenerate the VS files I'm not being able to due to an error that advises me the UnrealBuildTool can't find the Win64/shipping path in the BLU folder. I'm a bit confused on what the problem is since I read in the releases page for BLUI-3.1 that the shipping folder and moving those items were no longer needed so I'm not sure why the plugin is still looking for that path.

I have tried this with versions UE4 versions 4.8 through 4.11 Preview 8 along with trying engine source builds and launcher engine builds yet no go. And although I mentioned the release page I did try the master branch as well and get the same issues.

Visual Studio 2015 Regenerate Log: 
[VSRegenerateLog.txt](https://github.com/AaronShea/BLUI/files/194179/VSRegenerateLog.txt)

**Comment by AaronShea** [comment]
The shipping files are still very much needed. You should be downloading the VS2015 version from the release page and dropping that into your project's Plugins folder

**Comment by CJRobertson** [comment]
I got it working now. I made a few lame errors when I attempted before posting this as an issue.

First off, I misread a sentence in your 3.0 release where you were explaining you don't need to copy any files for a packaged game because the shipping folder is removed for you automatically. I read it as, "you don't have to do anything with the shipping folder in general" and applied this misthinking to the 3.1 release.. Go me for not paying attention.

Second, when I saw "Linux" in the download link I instantly thought it had something to do only with Linux projects which none of mine are at the moment. That wasn't the case and that folder had everything I needed. 

Thank you for the help even though it was just you pointing out I needed to pay closer attention, haha.

![blui_install_successful](https://cloud.githubusercontent.com/assets/13307644/14134578/371bf784-f60d-11e5-89e7-01b274576bdb.PNG)
